api/cotrip_app/models.py

```diff
@@ -1,3 +1,28 @@
 from django.db import models
 
+
+class User(models.Model):
+    email = models.EmailField(max_length=200)
+    password = models.CharField(max_length=100)
+
+    def __str__(self):
+        # return super().__str__()
+        return self.email
+
+
+class Profile(models.Model):
```

Comment:
Here is a really helpful article about extending the User model to a Profile model. Read through this - it may inspire you to make some adjustments on how you want to set this up. Instead of a one-to-many relationship, you will want the User and Profile to have a one-to-one relationship.

[How to Extend Django User Model](https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model)